test/integration/targets/groupby_filter/runme.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+MYTMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
+
+if [ -f /usr/bin/python3 ]
+then
+    PYTHON="--python /usr/bin/python3"
+else
+    PYTHON=""
+fi
+
+virtualenv --system-site-packages $PYTHON "${MYTMPDIR}/jinja2"
+
+source "${MYTMPDIR}/jinja2/bin/activate"
```

Comment:
I'll give it another shot.  I had it there, but it remember it failing on the osx build, which selected a different python than the venv that is used in the tests.  Just not setting `--python` in virtualenv seemed to fix it.